open-metadata-test/open-metadata-fvt/open-types-fvt/open-types-test/pom.xml

```diff
@@ -32,7 +32,7 @@
 
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
```

Comment:
needs checking. We had slf4j-simple here since in this module we RUN the test and so we need a runtime binding. #2089 made some changes in this area.

We may need slf4j-api - but that's compile time, and if the fvt code hasn't changed I can't see how we now need it. slf4j-simple is runtime, and we do need something.

Reply:
Closing this review comment - will track in https://github.com/odpi/egeria/issues/2265